doc/tools/kconfig/functions.rst

```diff
@@ -0,0 +1,28 @@
+.. _functions:
```

Comment:
label here should probably be more specific:  ``kconfig-functions`` (to avoid future label conflicts)